extensions/amp-story-shopping/0.1/amp-story-shopping-attachment.js

```diff
@@ -56,6 +61,12 @@ export class AmpStoryShoppingAttachment extends AMP.BaseElement {
 
     /** @private {!Map<string, Element>} */
     this.builtTemplates_ = {};
+
+    /** @private @const {!./story-analytics.StoryAnalyticsService} */
+    this.analyticsService_ = Services.storyAnalyticsService(this.win);
+
+    /** @protected {?../../amp-story/1.0/variable-service.AmpStoryVariableService} */
+    this.variableService_ = getVariableService(this.win);
```

Comment:
Also I believe you can do as above `Services.storyAnalyticsService` and use `Services.storyVariableService` because this class `dependsOnStoryServices` so it's guaranteed to be installed from the AmpStory constructor.